data/json/items/containers.json

```diff
@@ -474,24 +468,35 @@
     "looks_like": "box_medium",
     "description": "A very large cardboard box, the sort children would have loved to hide in, when there were still children.",
     "weight": "1250 g",
-    "volume": "3150 ml",
-    "//": "Volume is much lower than the actual volume of a box this size because presumably if it's in your pack, it isn't empty and full of air; and if it's in your hands, it's irrelevant what the volume is.",
+    "volume": "60100 ml",
+    "longest_side": "45 cm",
```

Comment:
These are deployable as hiding places, so they also need to be able to hide adults.

When I made it I was specifically looking at the box my dishwasher came in, which was about 90cm x 60cm x 60cm, and I, an average-sized ~~survivor~~ adult male could fit inside it (uncomfortably). That's a volume of around 320 L, and a longest_side of about 100cm.